linux-glibc/build_container/getenvoy_linux_glibc_bazel.sh

```diff
@@ -30,7 +30,10 @@ add-apt-repository \
    $(lsb_release -cs) \
    stable"
 
-apt-get update && apt-get -y install docker-ce docker-ce-cli containerd.io
+curl -fsSL https://apt.kitware.com/keys/kitware-archive-latest.asc | apt-key add -
+apt-add-repository "deb https://apt.kitware.com/ubuntu/ $(lsb_release -cs) main"
+
+apt-get update && apt-get -y install docker-ce docker-ce-cli containerd.io cmake
```

Comment:
nits: Is it possible to merge these lines to L23-25? https://github.com/tetratelabs/getenvoy-package/pull/96/files#diff-a1db49d1f80fa46ef76c7081ad0b378585ae5dcb9b7625a8bd46696889fa9040R23-R25 We might not need to care about the number of layers for this image, but just curious.

Reply:
No, `software-properties-common` needed for `apt-add-repository`, and this install depends on those repositories.